### DISCUSSION TITLE
New dataset for FineGray.R

### DISCUSSION THREAD

**Discussion by hminle** [Jeanselme/NeuralFineGray#4] [Q&A] [answered]
Hi @Jeanselme ,

I want to add my own dataset to run with FineGray.R
But seems that, it should be prepared in a special way.
When I run my own dataset with FineGray.R, it just shows this error:
<img width="842" alt="image" src="https://github.com/Jeanselme/NeuralFineGray/assets/16201681/ffb45613-90c9-4d2f-97a0-93a2ce00e34d">

Do you know what it is?

**Comment by hminle** [comment]
oh, I found out the reason why. It seems that the data should contain no column with all 0 values. Is that why you have to delete one column in SEER dataset?

**Reply by Jeanselme** [reply, marked as answer]
Yes. If columns are highly correlated, this may also lead to non-invertibility. In our experiments, we did not want to change the preprocessing for these two methods. However, one cousidering consider performing PCA as a solution.
